Hyperswitch-React-Demo-App/webpack.dev.js

```diff
@@ -8,11 +8,11 @@ let devServer = {
   port: 9060,
   historyApiFallback: true,
   proxy: {
-    "/payments": {
+    "/": {
```

Comment:
please remove this change cuz we have already fixed this and tested with docker setup
take a look at this PR for your reference

Reply:
<img width="692" alt="image" src="https://github.com/juspay/hyperswitch-web/assets/121166031/f18f82d9-80d7-4be1-91ee-a19ef8419d92">

The PR does not contain the change in the proxy. It is still hitting /payments, will that not cause any issue?

Reply:
It has to be /payments in both places, we had asked for the change to be made in  the CDK script itself.

Reply:
Okay, will revert this change